test/Evaluate/intrinsics.cpp

```diff
@@ -4,6 +4,7 @@
 #include "flang/Evaluate/expression.h"
 #include "flang/Evaluate/tools.h"
 #include "flang/Parser/provenance.h"
+#include "llvm/Support/raw_ostream.h"
 #include <initializer_list>
 #include <iostream>
```

Comment:
Use of this header is forbidden in LLVM. We should be able to remove it now?

Reply:
This isn't library code. It's a test program to generate tests.

Reply:
This file has this method:
 void DoCall(std::optional<DynamicType> resultType = std::nullopt, int rank = 0, bool isElemental = false)
That uses std::cout, as the compiler did not complain about it I did not change. For that reason also that this is there.

Reply:
I can do another commit removing all "#include <iostream>" in this PR if you all think it is fine. 
ATM this PR removes all uses of std::ostream in the code. I can also remove all the std::cout.

Reply:
Another point is that I needed to all llvm because of this method:
void DoCall that uses AsFortran method and all AsFortran method was changed to use llvm:raw_ostream